src/_ert/forward_model_runner/forward_model_step.py

```diff
@@ -85,35 +85,35 @@ class ForwardModelStep:
     MEMORY_POLL_PERIOD = 5  # Seconds between memory polls
 
     def __init__(
-        self, job_data: ForwardModelStepJSON, index: int, sleep_interval: int = 1
+        self, step_data: ForwardModelStepJSON, index: int, sleep_interval: int = 1
```

Comment:
Separate issue, but could consider calling this either just `step` or `step_json`. The postfix `_data` doesn't really add anything.